.github/workflows/ci.yml

```diff
@@ -4,42 +4,34 @@ on: [push]
 
 env:
   CI: true
-  DEFAULT_NODE_VERSION: 16.x
 jobs:
   build:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
```

Comment:
```suggestion
      - uses: actions/setup-node@v2
```

Reply:
Why this change? Does actions/setup-node automatically print out the version number? also, I think there's now setup-node@v3

Reply:
There is indeed a v3 and sure, we should use that.

I think we should in most cases (not sure if many justify the opposite) leave the standard action name as default instead of customising the messages, because:

- Yes the node version gets printed as well as all the parameters to the action
- More code/prose means more maintenance 
-  The denormalisation is actually bad because I want to see at a glance which standard action is being used
- In this specific case, why do we even improvise to say “Use” when the action says “Setup”?


In any case, I think we’re better off without this kind of prose and denormalisation.